assets/helpers/abTests/helpers.js

```diff
@@ -0,0 +1,49 @@
+// @flow
+
+import { tests as allTests } from 'helpers/abTests/abtestDefinitions';
+import { type AmountsRegions } from 'helpers/contributions';
+import type { Test, Participations } from 'helpers/abTests/abtest';
+
+function overrideAmountsForTest(
+  variantId: string,
+  abTest: Test,
+  currentAmountsRegions: AmountsRegions,
+): AmountsRegions {
+
+  const newAmountsRegions = { ...currentAmountsRegions };
+
+  const variant = abTest.variants.find(v => v.id === variantId);
+
+  if (variant && variant.amountsRegions) {
+    const { amountsRegions } = variant;
+
+    Object.keys(amountsRegions).forEach((countryGroupId) => {
+
+      Object.keys(amountsRegions[countryGroupId]).forEach((contributionType) => {
+
+        newAmountsRegions[countryGroupId][contributionType] = amountsRegions[countryGroupId][contributionType];
+      });
+    });
+  }
```

Comment:
maybe we could try this in future? https://www.npmjs.com/package/deepmerge

especially since we'll be doing something similar for Contribution Types and maybe other parameters

Reply:
in fact, I wonder if a more general solution would be to let a test override the entire `settings` object? (not necessarily for this PR, but as we extend what can be overridden by a test)

open to opinions on this. @rupertbates any thoughts?

Reply:
`deepmerge` looks good.

And good point - this could be very generic. Each variant could instead define a partial `Settings` object, which is used to 'patch' the current `settings`.